props/saber_fett263_buttons.h

```diff
@@ -45,12 +85,44 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if(!SaberBase::IsOn()) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+      // Edit '250' value in line below to change swing on sensitivity, 250 ~ 400 work best in testing
+        if (!swinging_ && fusor.swing_speed() > 250) {
+          swinging_ = true;
+          Event(BUTTON_NONE, EVENT_SWING);
+        }
+        if (swinging_ && fusor.swing_speed() < 100) {
+          swinging_ = false;
+        }
+      }
+    } 
     if (!swinging_ && fusor.swing_speed() > 250) {
       swinging_ = true;
       Event(BUTTON_NONE, EVENT_SWING);
-    }
+    }	  
+    if (auto_lockup_on_ && 
+        !swinging_ && 
+        fusor.swing_speed() > 120 && 
+        millis() - clash_impact_millis_ > FETT263_LOCKUP_DELAY &&
+        SaberBase::Lockup()) {
+          SaberBase::DoEndLockup();
```

Comment:
too much indentation, should be two more than the if on line 106 (same for the next two lines)